src/ResourceManager/AzureBatch/Commands.Batch/Models.Generated/PSVirtualMachineConfiguration.cs

```diff
@@ -34,18 +34,19 @@ public partial class PSVirtualMachineConfiguration
         
         internal Microsoft.Azure.Batch.VirtualMachineConfiguration omObject;
         
+        private PSContainerConfiguration containerConfiguration;
+        
+        private IList<PSDataDisk> dataDisks;
+        
         private PSImageReference imageReference;
         
+        private PSOSDisk oSDisk;
+        
         private PSWindowsConfiguration windowsConfiguration;
         
-        public PSVirtualMachineConfiguration(PSImageReference imageReference, string nodeAgentSkuId, PSWindowsConfiguration windowsConfiguration = default(PSWindowsConfiguration))
+        public PSVirtualMachineConfiguration(PSImageReference imageReference, string nodeAgentSkuId)
```

Comment:
since the original constructor isn't obsoleted, should we keep it?

Reply:
In this case I don't think so since this is a breaking change release, we should feel free to make this sort of change (even though I missed obsoleting it previously).